service/src/test/java/com/ge/predix/controller/test/SubjectPrivilegeManagementControllerIT.java

```diff
@@ -82,16 +85,36 @@
     private ConfigurableEnvironment env;
 
     @BeforeClass
-    public void setup() throws Exception {
+    public void setup() {
         this.testZone = TEST_UTILS.createTestZone("SubjectMgmtControllerIT");
         this.testZone2 = TEST_UTILS.createTestZone("SubjectMgmtControllerIT2");
         this.zoneService.upsertZone(this.testZone);
         this.zoneService.upsertZone(this.testZone2);
+    }
+
+    @BeforeMethod
```

Comment:
Refer to comments for PolicyManagementControllerIT regarding setup() and beforeMethod().  Setting up testZone2 could be deferred until the  test that is using it - testSameSubjectDifferentZones()